core/node/node_framework/src/implementations/layers/proof_api/mod.rs

```diff
@@ -53,13 +70,13 @@ impl MockStructLayer {
 
 impl MockStruct {
     pub fn new(
-        //blob_store: Arc<dyn ObjectStore>,
+        blob_store: Arc<dyn ObjectStore>,
```

Comment:
Can we remove this now, if we are not using this?

Reply:
Actually, we are using it to fetch proofs.